webhookclient/argus.go

```diff
@@ -116,30 +120,20 @@ func determineTokenAcquirer(config ArgusConfig) (acquire.Acquirer, error) {
 
 	return defaultAcquirer, nil
 }
-func createAttributeFilter(filter map[string]string) string {
-	if len(filter) == 0 {
-		return ""
-	}
-	buf := bytes.NewBufferString("?attributes=")
-
-	for key, value := range filter {
-		buf.WriteString(key + "," + value)
-		buf.WriteString(",")
-	}
-	buf.Truncate(buf.Len() - 1)
-	return buf.String()
-}
 
-func (c *ArgusClient) GetWebhook() ([]webhook.W, error) {
+func (c *ArgusClient) GetWebhook(owner string) ([]webhook.W, error) {
```

Comment:
I think you should remove any mention of argus in file/variable/function names from this webhookclient package.